src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 3 && days < 7) {
+    price -= 20;
+  } else if (days >= 7) {
+    price -= 50;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T07%3A07%3A40Z) 1st item. You should create variables for main values - 20, 50, 40 they shouldn't be just magic numbers

Reply:
Also, checklist 3rd item, you can just return values from your if statements instantly to avoid executing redundant code. And you don't really need else statements here in this way